app/code/Magento/MediaGallery/Test/Unit/Model/Asset/Command/GetByIdTest.php

```diff
@@ -0,0 +1,152 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MediaGallery\Test\Unit\Model\Asset\Command;
+
+use Magento\Framework\App\ResourceConnection;
+use Magento\Framework\DB\Adapter\AdapterInterface;
+use Magento\Framework\DB\Select;
+use Magento\Framework\Exception\IntegrationException;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager;
+use Magento\MediaGallery\Model\Asset\Command\GetById;
+use Magento\MediaGalleryApi\Api\Data\AssetInterface;
+use Magento\MediaGalleryApi\Api\Data\AssetInterfaceFactory;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Psr\Log\LoggerInterface;
+use Zend\Db\Adapter\Driver\Pdo\Statement;
+
+/**
+ * Test the GetById command model
+ * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
```

Comment:
@coderimus can you please remove this warning suppression (and extract some tests to a separate file/class if necessary)